webapp/packages/plugin-data-spreadsheet-new/src/DataGrid/useTableData.ts

```diff
@@ -66,16 +70,14 @@ export function useTableData(
       const rowStrings = this.format.getLongestCells();
 
       // TODO: seems better to do not measure container size
-      //       for detecting max columns size, better to use configurable variable
+      // for detecting max columns size, better to use configurable variable
       const measuredCells = TextTools.getWidth({
-        font: '400 14px Roboto',
-        text: columnNames.map((cell, i) => {
-          if (cell.length > (rowStrings[i] || '').length) {
-            return cell;
-          }
-          return rowStrings[i];
+        font: FONT,
+        text: columnNames.map((columnName, i) => {
+          const cellString = rowStrings[i] || '';
+          return columnName.length >= cellString.length ? columnName : cellString;
         }),
-      }).map(v => v + 16 + 32 + 20);
+      }).map(v => v + COLUMN_HEADER_CONTAINER_WIDTH + COLUMN_NAME_LEFT_MARGIN);
```

Comment:
offsets of column name and value are different, and offsets of column name are bigger.
we need to measure the size of the column's name and cell value separately and choose a bigger